rpa-home/modules/ROOT/pages/automation-userrolesandpermissions-privilegesbymodule.adoc

```diff
@@ -0,0 +1,358 @@
+= List of Privileges by Module
+
+RPA Manager has the following privileges, which can be assigned to individual users.
+
+To open any module, you always require the privilege for which the name ends in *Open*.
+
+Privileges that end with *Administration* authorize a user to create, change, and delete all types of objects in the corresponding module.
+If the authorization only applies to a single type of object in a module, the corresponding privileges end with  *Create*, *Edit*, and *Delete*.
+
+Privileges named *Global <...> View* authorize a user to see all the processes in the relevant view. A user who does not have this privilege can only see processes for which they belong to the project team.
+
+If you assign a role to a user, the user receives all the privileges contained in the role. These privileges are not displayed in the *Privileges* field in the user's profile.
+
+
+
+
+== Privileges of category *System*
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Logfile Download*
+|Download the log files from the server using the *Download logfiles* button in the *About* dialog
+
+|*Show Event Log*
+|Display the event log of the server using the *Open Event log* button in the *About* dialog
+
+|===
+
+
+
+
+== Privileges for the *User Management* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*User Management Open*
+|Open all views of the *User Management* module.
+
+|*User Management Administration*
+|Create, change, and delete all objects of the *User Management* module.
+
+|*User Api Key Create*
+|Create user API keys
+
+|*User Api Key Edit*
+|Change user API keys
+
+|*User Api Key Delete*
+|Delete user API keys
+
+|*User Group Create*
+|Create user groups
+
+|*User Group Edit*
+|Change user groups
+
+|*User Group Delete*
+|Delete user groups
+
+|===
+
+
+
+
+== Privileges of the *Bot Management* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Robot Management Open*
+|Open all views of the *Bot Management* module.
+
+|*Robot Management Administration*
+|Create, change, and delete all objects of the *Bot Management* module.
+
+|*Service Time Create*
+|Create bot service times
+
+|*Service Time Edit*
+|Change bot service times
+
+|*Service Time Delete*
+|Delete bot service times
+
+|===
+
+
+
+
+== Privileges of the *Process Evaluation* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Process Evaluation Open*
+|Open the *Process Evaluation* view
+
+|*Process Evaluation Administration*
+|Create, change, and delete all objects of the *Process Evaluation* module
+
+|*Global Process Evaluation View*
+|Authorization to view all processes in the views *Process Evaluation* and *My X1* &gt; *Team Backlog*, regardless of whether or not the current user belongs to the Process Team.
```

Comment:
It is called "My RPA" now

Reply:
This is also mentioned in the section about my rpa again. I s this on purpose?

Reply:
Yes, because the privilege affects both modules.